src/combinatorics/simple_graph/connectivity/subgraph.lean

```diff
@@ -0,0 +1,213 @@
+/-
+Copyright (c) 2023 Kyle Miller, Rémi Bottinelli. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller, Rémi Bottinelli
+-/
+
+import combinatorics.simple_graph.basic
+import combinatorics.simple_graph.connectivity
+import combinatorics.simple_graph.subgraph
+/-!
+# Connectivity of subgraphs
```

Comment:
```suggestion
# Connectivity of subgraphs and induced graphs
```

Could you also move everything that's specifically about induced graphs (not induced subgraphs) to a section at the end?

It would also be good to arrange other things more thematically, like putting `walk.to_subgraph_connected` right after its two supporting lemmas.

Reply:
> It would also be good to arrange other things more thematically, like putting `walk.to_subgraph_connected` right after its two supporting lemmas.

Not sure exactly how to do that, since `walk.to_subgraph_connected` the dependencies of lemmas aren't really linear starting with `.sup`, but I tried.